website/project/views/contributor.py

```diff
@@ -505,6 +507,29 @@ def send_claim_email(email, user, node, notify=True, throttle=24 * 3600):
     return to_addr
 
 
+@contributor_added.connect
+def notify_contributor(node, contributor, throttle=24 * 3600):
```

Comment:
When you do this, make sure to do:

```python
def notify_contributor(..., throttle=None):
    throttle = throttle or settings.CONTRIBUTOR_ADDED_EMAIL_THROTTLE
```

This will allow the setting to be changed at runtime, which is useful for testing. 

Speaking of which, can you add a test to make sure the throttling works?